contracts/0.8.25/vaults/VaultHub.sol

```diff
@@ -334,6 +381,35 @@ abstract contract VaultHub is PausableUntilWithRoles {
         emit VaultRebalanced(msg.sender, sharesToBurn);
     }
 
+    /// @notice forces validator withdrawal from the beacon chain in case the vault is unbalanced
+    /// @param _vault vault address
+    /// @param _pubkeys pubkeys of the validators to withdraw
+    /// @param _refundRecepient address of the recipient of the refund
+    function forceValidatorWithdrawal(
```

Comment:
still trigger but not force?

Reply:
I'm not sure I got your comment right. Wdym?